src/components/MovieReviewItem/index.js

```diff
@@ -25,20 +31,43 @@ const MovieReviewItem = ({
             {/*        <MovieItem movie={review.movie} posterOnClickHandler={() => {}}/>*/}
             {/*    </div>*/}
             {/*}*/}
-            <div className="col-12 card border-0 bg-light">
-                <div className="card-body d-flex justify-content-between">
-                    <div>
-                        <h5 className="card-title p-1">User: {review.postedBy.username}</h5>
-                        <h6 className="card-subtitle text-muted p-1 ">Rating: {review.rating}</h6>
-                        <p className="card-text p-1">{review.review}</p>
-                    </div>
-                    <div>
+            <div className="col-12 border-0 bg-light">
+                <div className="">
+                {location.pathname === '/profile/s/reviews' && 
+                <div className="row p-4" onClick={() => onClickNavigate(review.movie)}>
+                        <div className="col-2 col-sm-3 col-md-3 col-lg-3 col-xxl-2">
+                            <img src={`${review.movie.poster_path ? `${process.env.REACT_APP_MOVIE_BASE_URL}/w342/${review.movie.poster_path}` : ""}`} className="img-fluid" alt="Poster Not Found" />
+                        </div>
+                        <div className="col-9 col-sm-6 col-md-7 col-lg-8 col-xxl-9">
+                        <h5 className=" p-1">{review.movie.original_title}</h5> 
+                        <h6 className=" text-muted p-1 ">Rating: {review.rating}</h6>
+                        <p className=" p-1">{review.review}</p>
+                        </div>
+                        <div className="col-1 col-sm-3 col-md-2 col-lg-1 col-xxl-1">
                         {
                             isMyReview &&
                             <button className={"btn btn-danger"}
                                     onClick={deleteReview}>Delete</button>
                         }
+                        </div>
                     </div>
+                    }
+                    {location.pathname !== '/profile/s/reviews' && <div className="row p-2">
```

Comment:
Actually, now we basically have 2 versions of the movie review item, I think it's better we create one component for each version instead of having this path check here to decide.

Reply:
Can you suggest on how to refeactor this over a call today?

Reply:
Create a new component, and move lines 49-63 to there. Then copy-paste some variables to make it work.